kasa/discover.py

```diff
@@ -331,20 +340,25 @@ async def connect_single(
         The device type is discovered by querying the device.
```

Comment:
Please modify the docstring to state that passing `device_type` *forces* the type and avoids the discovery.

Reply:
adjusted in bcdcd5f982bfccf39c33c03300ebdaeb88d8dd09